File: server/LotteryApi/Models/PackageModel.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;
using System.ComponentModel.DataAnnotations;

namespace LotteryApi.Models
{
    public class PackageModel : ITenantEntity
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public string Id { get; set; } = null!;

        [Required]
        public string Name { get; set; }

        [Required]
        public int Price { get; set; }

        [Required]
        public int QtyClassicCards { get; set; }

        [Required]
        public int QtySpecialCards { get; set; }

        [Required]
        public int QtyPrimumCards { get; set; }

        public string OrganizationId { get; set; } = null!;
    }
}

File: client/src/models/package.model.ts
export interface PackageDto {
    id: number;
    name: string;
    price: number;
    qtyClassicCards: number;
    qtySpecialCards: number;
    qtyPrimumCards: number;
}

export interface PackageCreateDto {
    name: string;
    price: number;
    qtyClassicCards: number;
    qtySpecialCards: number;
    qtyPrimumCards: number;
}

export interface PackageUpdateDto {
    name?: string;
    price?: number;
    qtyClassicCards?: number;
    qtySpecialCards?: number;
    qtyPrimumCards?: number;
}